tools/test_certs.sh

```diff
@@ -4,63 +4,71 @@ set -e
 
 OUT_DIR=$1      # Directory to write the new certificates to
 TOOL=$2         # The certificate generation tool
-IMAGE_TAG=$3    # The docker image tag to use
+DNS_NAME=$3     # The DNS name to use to connect the client and server
+IMAGE_TAG=$4    # Optional Docker image tag used in terraform
 
-ps aux | grep management | awk '{print $2}' | xargs kill || true
-
-# Clean up will prompt the user to clean
-# up any resources created during the script
-function cleanUp {
-    # echo "Ion management API running as PID: $MGMT_PID"
-    # echo "want me to kill it? (y/n)"
-    # read killIt
-    # if [[ "$killIt" == "y" || "$killIt" == "Y" ]]; then
-        kill $MGMT_PID || true
-    # fi
-
-    # echo "This script generated the following files:"
-    # echo " - management"
-    # echo " - ioncli"
-    # echo "Do you want me to remove them? (y/n)"
-    # echo "want me to kill it? (y/n)"
-    # read deleteThem
-    # if [[ "$deleteThem" == "y" || "$deleteThem" == "Y" ]]; then
-        rm management || true
-        rm ioncli || true
-    # fi
-}
-
-trap cleanUp EXIT
+if pgrep management; then pkill -9 management &> /dev/null; fi
+if pgrep ioncli; then pkill -9 ioncli &> /dev/null; fi
 
 echo "------------------------------------"
 echo "Generating new certificates using: $TOOL"
 echo "------------------------------------"
 case "$TOOL" in
-     "openssl")
-          ./tools/generate_certs_openssl.sh $OUT_DIR
-          ;;
-     "terraform")
-          ./tools/generate_certs_terraform.sh $OUT_DIR $IMAGE_TAG
-          ;;
-     *)
-          echo "Unrecognized tool choice:  $TOOL"
-          echo "Options are: openssl, terraform"
-          exit
-          ;;
-esac
+    "openssl")
+        if [ -z "$DNS_NAME" ];then
+            DNS_NAME="localhost"
+        fi
+        echo "I will generate new certificates"
+        echo "I will execute a new deployment. Please be patient"
+        echo "this can take up to 30 minutes."
```

Comment:
Does this line make sense here look like it may be copy paste from terraform section